src/Extensions.Base.Api/BeginCallbacks.cs

```diff
@@ -45,9 +45,11 @@ internal class BCB<T>
 
 }
 
-//Underscore _System assembly will be treated as regular System namespace by ImportModule function of Intstrumentation Engine.
+//Underscore _System assembly will be treated as regular System namespace by ImportModule function of Instrumentation Engine.
 //This is done to avoid conflicts during compilation
+#pragma warning disable CS3008 // Identifier is not CLS-compliant
 namespace _System.Diagnostics
```

Comment:
So, in other words, the assembly is *not* CLS compliant. So, we shouldn't really mark it as-such should we?

Reply:
Good point, I'll remote the pragma warning